Bluepill-cli/Bluepill-cli/Simulator/SimulatorHelper.m

```diff
@@ -12,9 +12,47 @@
 #import "BPConfiguration.h"
 #import "BPUtils.h"
 #import "BPXCTestFile.h"
+#import "BPSimulator.h"
+#import "BPTestClass.h"
 
 @implementation SimulatorHelper
 
++ (NSDictionary *)createSimTemplatesAndDumpTests:(NSArray<BPXCTestFile *> *)xcTestFiles withConfig:(BPConfiguration *)config {
+    BPSimulator *bpSimulator = [BPSimulator simulatorWithConfiguration:config];
+    NSDictionary *testHostSimTemplates = [bpSimulator createSimulatorAndInstallAppWithBundles:xcTestFiles];
+    assert([testHostSimTemplates count] > 0);
```

Comment:
An `assert()` here seems too harsh. If there are failures launching the app and getting the tests I think we need to handle the error.